ethclient/gethclient/gethclient.go

```diff
@@ -204,6 +219,46 @@ func (ec *Client) SubscribePendingTransactions(ctx context.Context, ch chan<- co
 	return ec.c.EthSubscribe(ctx, ch, "newPendingTransactions")
 }
 
+// TraceCall lets you trace a given eth_call
+func (ec *Client) TraceCall(ctx context.Context, msg ethereum.CallMsg, blockNrOrHash rpc.BlockNumberOrHash, config *tracers.TraceCallConfig) (interface{}, error) {
```

Comment:
@fjl What do you think about using `rpc.BlockNumberOrHash` here? Normally we use `*big.Int` but TraceCall can accept both number or hash. Would you say it's justified or rather 2 separate methods?

Reply:
@fjl  Hi, what's your opinion?

Reply:
I think it's ok. Resolving this comment.